ext/flash/lib.rs

```diff
@@ -150,13 +150,52 @@ async fn op_flash_respond_async(
   Ok(())
 }
 
+#[op(fast)]
+fn op_try_flash_respond_chuncked(
+  op_state: &mut OpState,
+  server_id: u32,
+  token: u32,
+  response: &[u8],
+  shutdown: bool,
+) -> u32 {
+  let flash_ctx = op_state.borrow_mut::<FlashContext>();
+  let ctx = flash_ctx.servers.get_mut(&server_id).unwrap();
+  let tx = ctx.requests.get(&token).unwrap();
+  let sock = tx.socket();
+
+  let mut n = 0;
+  // TODO(@littledivy): Use writev when `UnixIoSlice` lands.
+  // https://github.com/denoland/deno/pull/15629
+  macro_rules! try_write_oops_bye {
+    ($e:expr) => {
+      let e = $e;
+      let expected = sock.try_write(e);
+      n += expected;
+      if expected != e.len() {
+        return n as u32;
+      }
+    };
+  }
+
+  let h = format!("{:x}\r\n", response.len());
```

Comment:
Consider using `write!()` on a `&mut [u8]`, that saves a `String` heap allocation.